app/test/prepare_proposal_test.go

```diff
@@ -30,6 +33,74 @@ import (
 	blobtypes "github.com/celestiaorg/celestia-app/v4/x/blob/types"
 )
 
+func TestPrepareProposalValidConstruction(t *testing.T) {
+	// Reproduces https://github.com/celestiaorg/celestia-app/issues/4961
+	t.Run("prepare proposal creates a proposal that process proposal throws an account sequence mismatch", func(t *testing.T) {
```

Comment:
[nit] now that the test passes 🎊 

```suggestion
	t.Run("prepare proposal creates a proposal that process proposal accepts", func(t *testing.T) {
```